user/utils.py

```diff
@@ -0,0 +1,22 @@
+import jwt
+import json
+
+from django.http        import JsonResponse
+from user.models        import User
+from my_settings        import SECRET_KEY, ALGORITHM
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
```

Comment:
`*args, **kwargs`를 인자로 받는 이유는 무엇인가요?
```suggestion
    def wrapper(self, request, *args, **kwargs):
```

Reply:
이부분 개념이 부족한것같습니다. 조금 더 찾아보고 정리해서 답글 남기도록하겠습니다.

Reply:
여기 병민님이 말씀해주신 `*args`랑 `**kwargs` 왜 사용하는지 정리 되셨나요 규보님?